src/components/Filter/Filter.tsx

```diff
@@ -23,14 +21,60 @@ export default function BasicSelect(props: IFilter) {
   const { title, items, isSearchFilter } = props;
   const [selectedItem, setSelectedItem] = useState("");
 
+  useEffect(()=>{
+    if(isSearchFilter){
+      setSelectedItem(()=> Endpoint.TopHeadlines);
```

Comment:
I think that if you already have redux store, do not use local states, but its yours choise